Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -8372,6 +8377,8 @@ class BBQJIT {
     RegisterSet m_callerSaves;
 
     InternalFunction* m_compilation;
+
+    std::array<Vector<Jump>, numberOfExceptionTypes> m_exceptions { };
```

Comment:
Why not just use a `JumpList` instead of a `Vector<Jump>`?

Reply:
Right, we should use JumpList. Changed.